src/containers/pages/AssigmentMapWrapper/index.tsx

```diff
@@ -166,14 +166,14 @@ const AssignmentMapWrapper = (props: AssignmentMapWrapperProps) => {
     }
   }
 
-  if (!hasValidGeoms) {
-    return <div>{MAP_LOAD_ERROR}</div>;
-  }
-
   if (loading) {
     return <Loading />;
   }
 
+  if (!hasValidGeoms || !structures.length) {
```

Comment:
I don't think we should prevent the map from loading if we have no structures.  For example, it is pretty common in IRS reports to only have the boundary (jurisdiction) but no structures.  In which case I would like the map to load and just show the boundary.

I think we can take this idea further and aim for the map being able to load as long as we have any valid layers.

Does that sound like it could work?

Reply:
For this case we are geting the boundaries(map bounds and center) from the jurisdiction features. So unless the bounds are pre defined in settings then im not really sure if that would work.

Reply:
Where do we get structures in this component?

To answer your question, I made a comment [here](https://github.com/onaio/reveal-frontend/issues/1205#issuecomment-681899700).

Reply:
Got it, so i have added [this](https://github.com/onaio/reveal-frontend/pull/1194/commits/287308df0f830a874ad9186baa6205a9bb20f230) so that we can show an error message if features/layers are empty